src/System.Console/src/System/Console.cs

```diff
@@ -81,12 +82,12 @@ public static Encoding OutputEncoding
                     // s_out reinitialize the console code page.
                     if (Volatile.Read(ref s_out) != null && !s_isOutTextWriterRedirected)
                     {
-                        s_out.Flush();
+                        s_out!.Flush();
```

Comment:
This `!` is correct, but a little unfortunate.  @jcouv, our use of [NotNullIfNotNull] on Volatile.Read was intended to cover cases like this, but it falls short because it's only one directional: the compiler can know that the return value is not null if s_out is not null, but it can't then also infer that s_out is not null if the return value is not null.